src/editor/CodeHintList.js

```diff
@@ -408,7 +409,7 @@ define(function (require, exports, module) {
             } else if (keyCode === KeyEvent.DOM_VK_UP) {
                 _rotateSelection.call(this, -1);
             } else if (keyCode === KeyEvent.DOM_VK_DOWN ||
-                    (event.ctrlKey && keyCode === KeyEvent.DOM_VK_SPACE)) {
+                    (event.ctrlKey && keyCode === KeyEvent.DOM_VK_SPACE) || isFakeCallUp) {
```

Comment:
Do we need to touch this condition? In case of a fake key down for Ctrl+Space `(event.ctrlKey && keyCode === KeyEvent.DOM_VK_SPACE)` will return true anyways.